connector/connect/common/src/test/resources/query-tests/explain-results/function_array_append.explain

```diff
@@ -1,2 +1,2 @@
-Project [array_append(e#0, 1) AS array_append(e, 1)#0]
+Project [array_insert(e#0, (size(e#0, false) + 1), 1) AS array_append(e, 1)#0]
```

Comment:
You can fix `array_insert` to have `getTagValue(FunctionRegistry.FUNC_ALIAS).getOrElse(name)`, and set the alias in `FunctionRegistry` by setting `setAlias` to `true`

Reply:
Yeah. I forgot it. Thank you for your reminder.

Reply:
@HyukjinKwon I tested and found this can't given help.

Reply:
There are a lot golden files are the same as the case.
Refer: https://github.com/apache/spark/blob/f6d6ec138fe104bc7317993fe90bfdc672c11938/connector/connect/common/src/test/resources/query-tests/explain-results/function_map_contains_key.explain